elasticache/memcached/Acornfile

```diff
@@ -0,0 +1,121 @@
+name:        "AWS Elasticache Memcached"
+description: "Amazon's Memcached compatible Elasticache cluster"
+info:        "\(localData.info)"
```

Comment:
Can we get an icon? There's amazon assets someplace with little icons.

Reply:
Added icons for both Memcached and Redis